monai/data/image_reader.py

```diff
@@ -251,52 +254,54 @@ def verify_suffix(self, filename: Union[Sequence[str], str]) -> bool:
         suffixes: Sequence[str] = ["nii", "nii.gz"]
         return is_supported_format(filename, suffixes)
 
-    def read(self, data: Union[Sequence[str], str, Any]):
+    def read(self, data: Union[Sequence[str], str], **kwargs):
         """
-        Read image data from specified file or files, or set a Nibabel Image object.
+        Read image data from specified file or files.
         Note that the returned object is Nibabel image object or list of Nibabel image objects.
-        `self._img` is always a list, even only has 1 image.
 
         Args:
             data: file name or a list of file names to read.
+            kwargs: additional args for `nibabel.load` API, if not set, using `self.kwargs`.
+                More details about available args:
+                https://github.com/nipy/nibabel/blob/master/nibabel/loadsave.py
 
         """
-        self._img = list()
-        if isinstance(data, Nifti1Image):
-            self._img.append(data)
-            return data
+        img_: List[Nifti1Image] = list()
 
         filenames: Sequence[str] = ensure_tuple(data)
+        if len(kwargs) == 0:
+            kwargs = self.kwargs
         for name in filenames:
-            img = nib.load(name, **self.kwargs)
+            img = nib.load(name, **kwargs)
```

Comment:
would be great to have a test case with some non-empty kwargs

Reply:
added in the latest commit.
Thanks.